bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HPlusHF/HPlusCharm_3FS_MuRFScaleDynX0p50_HToGG_M125_TuneCP5_13p6TeV_amcatnlo_pythia8/HPlusCharm_3FS_MuRFScaleDynX0p50_HToGG_M125_TuneCP5_13p6TeV_amcatnlo_pythia8_run_card.dat

```diff
@@ -0,0 +1,170 @@
+               ! number of points and iter. below)
+# These numbers are ignored except if req_acc_FO is equal to -1
+  5000 = npoints_fo_grid ! number of points to setup grids
+  4 = niters_fo_grid ! number of iter. to setup grids
+  10000 = npoints_fo ! number of points to compute Xsec
+  6 = niters_fo ! number of iter. to compute Xsec
+#***********************************************************************
+# Random number seed                                                   *
+#***********************************************************************
+  0 = iseed ! rnd seed (0=assigned automatically=default)
+#***********************************************************************
+# Collider type and energy                                             *
+#***********************************************************************
+  1 = lpp1 ! beam 1 type (0 = no PDF)
+  1 = lpp2 ! beam 2 type (0 = no PDF)
+  6800.0 = ebeam1 ! beam 1 energy in GeV
+  6800.0 = ebeam2 ! beam 2 energy in GeV
+#***********************************************************************
+# PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
+#***********************************************************************
+  'lhapdf'             = pdlabel  ! PDF set
+  $DEFAULT_PDF_SETS    = lhaid    ! if pdlabel=lhapdf, this is the lhapdf number
+  $DEFAULT_PDF_MEMBERS = reweight_PDF
+#***********************************************************************
+# Include the NLO Monte Carlo subtr. terms for the following parton    *
+# shower (HERWIG6 | HERWIGPP | PYTHIA6Q | PYTHIA6PT | PYTHIA8)         *
+# WARNING: PYTHIA6PT works only for processes without FSR!!!!          *
+#***********************************************************************
+  PYTHIA8  = parton_shower 
+  0.5      = shower_scale_factor ! multiply default shower starting
+                                 ! scale by this factor
+#***********************************************************************
+# Renormalization and factorization scales                             *
+# (Default functional form for the non-fixed scales is the sum of      *
+# the transverse masses divided by two of all final state particles    * 
+# and partons. This can be changed in SubProcesses/set_scales.f or via *
+# dynamical_scale_choice option)                                       *
+#***********************************************************************
+  False = fixed_ren_scale ! if .true. use fixed ren scale
+  False = fixed_fac_scale ! if .true. use fixed fac scale
+ 32.0 = mur_ref_fixed ! fixed ren reference scale 
+ 32.0 = muf_ref_fixed ! fixed fact reference scale
+   -1 = dynamical_scale_choice ! Choose one (or more) of the predefined
+           ! dynamical choices. Can be a list; scale choices beyond the
+           ! first are included via reweighting
+  0.5 = mur_over_ref ! ratio of current muR over reference muR
+  0.5 = muf_over_ref ! ratio of current muF over reference muF
```

Comment:
do you want to use fixed ren/fac scale

Reply:
No, we're planning to use dynamic scale, as stated in the name of the data cards, the value reported here for the fixed scale parameter was used during previous tests and left for completeness.